packages/twenty-server/src/core/messaging/timeline-messaging.resolver.ts

```diff
@@ -0,0 +1,77 @@
+import { Args, Query, Field, Resolver, ObjectType } from '@nestjs/graphql';
+import { UseGuards } from '@nestjs/common';
+
+import { Column, Entity } from 'typeorm';
+
+import { JwtAuthGuard } from 'src/guards/jwt.auth.guard';
+import { Workspace } from 'src/core/workspace/workspace.entity';
+import { AuthWorkspace } from 'src/decorators/auth-workspace.decorator';
+import { TimelineMessagingService } from 'src/core/messaging/timeline-messaging.service';
+
+@Entity({ name: 'timelineMessage', schema: 'core' })
+@ObjectType('TimelineMessage')
+class TimelineMessage {
```

Comment:
nit: Ideally we should separate the graphql definition and the typeorm entity definition into 2 files. You can check object-metadata for example.